packages/statemanager/src/stateManager.ts

```diff
@@ -405,28 +405,40 @@ export class DefaultStateManager implements EVMStateManagerInterface {
     const contractCode = await this.getContractCode(address)
     return contractCode.length
   }
+
   /**
-   * Gets the storage trie for an account from the storage
-   * cache or does a lookup.
+   * Gets the storage trie for the EVM-internal account identified by the provided address/hash.
+   * If the storage trie is not in the local cache ('this._storageTries'),
+   *   generates a new storage trie object based on a lookup (shallow copy from 'this._trie'),
+   *   applies the storage root of the provided rootAccount (or an
+   *   empty trie root if no rootAccount is provided), and stores the new entry
+   *   in the local cache.
+   * @param addressOrHash Address (or other object) with populated 'bytes', or a raw Uint8Array.
+   *   Used to identify the requested storage trie in the local cache and define the
+   *   prefix used when creating a new storage trie.
+   * @param  rootAccount (Optional) Account object whose 'storageRoot' is to be used as
+   *   the root of the new storageTrie returned when there is no pre-existing trie.
+   *   If left undefined, the EMPTY_TRIE_ROOT will be used as the root instead.
+   * @returns storage Trie object
    * @private
    */
   // TODO PR: have a better interface for hashed address pull?
-  protected _getStorageTrie(addressOrHash: Address | Uint8Array, account?: Account): Trie {
+  protected _getStorageTrie(
+    addressOrHash: Address | { bytes: Uint8Array } | Uint8Array,
```

Comment:
Why add this `{ bytes: Uint8Array }` to the type union since it breaks the build on CI.  Does this add some benefit in the specific usecase your PR addresses related to `@ethereumjs/statemanager` as a sub-dependency?

Reply:
Thanks, will take another look!